pkg/updatestatus/updatestatuscontroller.go

```diff
@@ -29,6 +30,10 @@ import (
 // we have the Status API we need to serialize ourselves anyway.
 type informerMsg struct {
 	informer string
+	// knownInsight contains the UIDs of insights known by the informer, so the controller can remove insights formerly
+	// reported by the informer but no longer known to it (e.g. because the informer was restarted and the culprit
+	// condition ceased to exist in the meantime).
+	knownInsights []string
```

Comment:
We should add a comment that the `knownInsights` of a `message` do not contain the `uid` of the message. Something along these lines.

Reply:
Addressed in 0dfbb5cc7dfac9165d93bf9e38e3e6f8bab06530